.circleci/config.yml

```diff
@@ -140,8 +140,8 @@ jobs:
           at: ~/happa
 
       - run:
-          name: Validate code style using prettier
-          command: yarn run validate-prettier --no-color
+          name: Lint code using ESlint
+          command: yarn run lint --no-color
```

Comment:
```suggestion
          command: yarn run lint
```

Reply:
I don't think colors are supported in the CI output. Similarly to how we do that for prettier.. any reason for this suggestion?

Reply:
Thought you forgot it there, as it wasn't in the original lint CI command
But I see that ESLint also has [that flag](https://eslint.org/docs/user-guide/command-line-interface), so it's fine 👍